sycl/include/CL/sycl/INTEL/fpga_reg.hpp

```diff
@@ -26,10 +31,3 @@ template <typename _T> _T fpga_reg(const _T &t) {
 } // namespace sycl
 } // __SYCL_INLINE_NAMESPACE(cl)
 
-// Keep it consistent with FPGA attributes like intelfpga::memory()
-// Currently clang does not support nested namespace for attributes
-namespace intelfpga {
```

Comment:
This change removes the old API?  An API breaking change is supposed to require approval from the oneAPI PXT, and a change in the library versioning.  Why do we want to do this?

Reply:
I can leave this in but this API was never documented, and we moved to the INTEL::fpga_reg API as of gold. This is a residue from the older intel:: and intelfpga:: headers.

Reply:
I don't think we should remove it without asking approval at onAPI pxt.  I think it would be fine to remove, but if we were to get a customer breakage from this, I would want to be sure we didn't do this without having received approval.

Reply:
LGTM